src/current/App.jsx

```diff
@@ -54,308 +54,314 @@ const queryClient = new QueryClient();
+                  ),
+                  children: [
+                    {
+                      path: '/weekend-schedules',
+                      element: <WeekendMeetingSchedule />,
+                    },
+                  ],
+                },
+              ],
+            },
+            {
+              path: '/congregation-settings',
+              element: <CongregationSettings />,
+            },
+            {
+              element: <PrivateVipConnectedRoute isCongAccountConnected={isCongAccountConnected} isAdmin={adminRole} />,
+              children: [
+                {
+                  path: '/administration',
+                  element: <Administration />,
+                },
+                {
+                  path: '/administration/members/:id',
+                  element: <CongregationPersonDetails />,
+                },
+              ],
+            },
+          ],
+        },
+        { path: '*', element: <DashboardMenu /> },
+      ],
+    },
+  ]);
 
-	useEffect(() => {
-		if (isLight) {
-			setActiveTheme(lightTheme);
-		} else {
-			setActiveTheme(darkTheme);
-		}
-	}, [isLight]);
+  useEffect(() => {
+    if (isLight) {
+      setActiveTheme(lightTheme);
+    } else {
+      setActiveTheme(darkTheme);
+    }
+  }, [isLight]);
 
-	useEffect(() => {
-		const getUserID = async () => {
-			try {
-				const fp = await FingerprintJS.load();
-				const result = await fp.get();
+  useEffect(() => {
+    const getUserID = async () => {
+      try {
+        const fp = await FingerprintJS.load();
+        const result = await fp.get();
 
-				const visitorId = result.visitorId;
-				setVisitorID(visitorId);
-				backupWorkerInstance.setVisitorID(visitorId);
+        const visitorId = result.visitorId;
+        setVisitorID(visitorId);
+        backupWorkerInstance.setVisitorID(visitorId);
 
-				console.info('CPE: Fingerprint: Device visitor id has been set');
-			} catch (error) {
-				console.error(`CPE: Fingerprint: ${error}`);
-			}
-		};
+        console.info('CPE: Fingerprint: Device visitor id has been set');
+      } catch (error) {
+        console.error(`CPE: Fingerprint: ${error}`);
+      }
+    };
 
-		if (isOnline) getUserID();
-	}, [setVisitorID, isOnline]);
+    if (isOnline) getUserID();
+  }, [setVisitorID, isOnline]);
 
-	useEffect(() => {
-		let apiHost;
+  useEffect(() => {
+    let apiHost;
 
-		if (import.meta.env.VITE_BACKEND_API) {
-			apiHost = import.meta.env.VITE_BACKEND_API;
-		} else {
-			if (import.meta.env.DEV || window.location.host.indexOf('localhost') !== -1) {
-				apiHost = 'http://localhost:8000/';
-			} else {
-				apiHost = 'https://api.sws2apps.com/';
-			}
-		}
+    if (import.meta.env.VITE_BACKEND_API) {
+      apiHost = import.meta.env.VITE_BACKEND_API;
+    } else {
+      if (import.meta.env.DEV || window.location.host.indexOf('localhost') !== -1) {
+        if (import.meta.env.API_HOST_PORT) {
```

Comment:
@Logan08691: I think this proposed can be reverted. 😊 The CPE app should no longer need such customization, I guess...